src/components/Home/index.tsx

```diff
@@ -59,25 +31,16 @@ export default function HomePage(): ReactElement {
         <h3>Your Bookmarks</h3>
         <Bookmarks />
       </section>
-
-      <Allocations />
-
-      <SectionQueryResult
-        title="Highest veOCEAN Allocations"
-        query={queryMostAllocation}
-      />
-
-      <SectionQueryResult title="Most Sales" query={queryMostSales} />
-      <MostViews />
-      <TopSales title="Publishers With Most Sales" />
-      <TopTags title="Top Tags By Sales" />
-
```

Comment:
issue: I think some of these queries could actually also be interesting for us. I would suggest removing everything veOcean related (all "allocation" stuff) but keeping the other sections and going off of team/user feedback from there.